recipes/conda-index/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "conda-index" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/conda/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 07ac9ed0938b88a02f093644e4ab0fa41919fec87fa2e1b2f29a761a6e3f2953
+
+build:
+  script: {{ PYTHON }} -m pip install --no-build-isolation . -vv
+  number: 0
+  noarch: python
+  entry_points:
+    - conda-index = conda_index.cli:cli
+
+requirements:
+  host:
+    - flit-core
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - click >=8
+    - conda >=4.12.0
+    - conda-package-streaming
+    - filelock
+    - jinja2
+    - more-itertools
+    - PyYAML >=6
+  run_constrained:
+    # define conflict blocker for conda-build
+    # https://conda-forge.org/docs/maintainer/adding_pkgs.html#defining-conflicts
+    - conda-build ==9999999999
```

Comment:
```suggestion
```

They should be installed at the same time. Do we need to remove the console script from conda-index or rename?